src/OpenLoco/Vehicles/OrderSkip.cpp

```diff
@@ -0,0 +1,53 @@
+#include "../GameCommands.h"
+#include "../Interop/Interop.hpp"
+#include "../Things/ThingManager.h"
+#include "../Ui/WindowManager.h"
+#include "Orders.h"
+#include "Vehicle.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::Vehicles
+{
+    loco_global<uint16_t, 0x009C68E0> gameCommandMapX;
+    loco_global<uint16_t, 0x009C68E2> gameCommandMapY;
+    loco_global<uint16_t, 0x009C68E4> gameCommandMapZ;
+
+    // 0x0047071A
+    static uint32_t orderSkip(uint16_t headId, uint8_t flags)
+    {
+        auto* head = ThingManager::get<Vehicles::VehicleHead>(headId);
+        if (head == nullptr)
+        {
+            return GameCommands::FAILURE;
```

Comment:
You would never throw an exception from a game command as it could break the server when connected as a client. As for the uppercase that's just what it was defined as back when the game command runner was implemented